CMakeLists.txt

```diff
@@ -87,7 +90,13 @@ target_link_libraries(evmone PUBLIC evmc intx::intx PRIVATE ethash::keccak)
 add_subdirectory(silkworm)
 
 add_executable(check_changes cmd/check_changes.cpp)
-target_link_libraries(check_changes PRIVATE silkworm absl::flags_parse absl::time)
+
+if(MSVC)
+  # lmdb requires link to ntdll.lib
```

Comment:
Let's link silkworm to ntdll rather than check_changes and tests individually

Reply:
Wilco